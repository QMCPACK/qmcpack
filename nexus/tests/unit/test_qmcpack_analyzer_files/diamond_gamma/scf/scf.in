&CONTROL
   calculation     = 'scf'
   outdir          = 'pwscf_output'
   prefix          = 'pwscf'
   pseudo_dir      = './'
   wf_collect      = .true.
/

&SYSTEM
   ecutwfc         = 200
   ibrav           = 0
   input_dft       = 'lda'
   nat             = 2
   nosym           = .true.
   nspin           = 1
   ntyp            = 1
   tot_charge      = 0
/

&ELECTRONS
   conv_thr        = 1e-08
/


ATOMIC_SPECIES 
   C  12.011 C.BFD.upf

ATOMIC_POSITIONS bohr
   C        0.00000000       0.00000000       0.00000000 
   C        1.68658058       1.68658058       1.68658057 

K_POINTS automatic
   1 1 1  0 0 0 

CELL_PARAMETERS bohr
         3.37316115       3.37316115      -0.00000000 
         0.00000000       3.37316115       3.37316115 
         3.37316115       0.00000000       3.37316115 


