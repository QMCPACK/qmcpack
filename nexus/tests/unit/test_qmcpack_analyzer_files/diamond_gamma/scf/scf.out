
     Program PWSCF v.6.3MaX starts on  5Nov2019 at  9:20:28 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on    12 processors

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =      12
     Reading input from scf.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = LDA ( 1  1  0  0 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file C.BFD.upf: wavefunction(s)  0s 0p renormalized

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         105     105     26                 2446     2446     307
     Max         106     106     27                 2449     2449     310
     Sum        1261    1261    313                29363    29363    3695



     bravais-lattice index     =            0
     lattice parameter (alat)  =       4.7704  a.u.
     unit-cell volume          =      76.7611 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=            4
     kinetic-energy cutoff     =     200.0000  Ry
     charge density cutoff     =     800.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = LDA ( 1  1  0  0 0 0)

     celldm(1)=   4.770370  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.707107   0.707107  -0.000000 )  
               a(2) = (   0.000000   0.707107   0.707107 )  
               a(3) = (   0.707107   0.000000   0.707107 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  0.707107  0.707107 -0.707107 )  
               b(2) = ( -0.707107  0.707107  0.707107 )  
               b(3) = (  0.707107 -0.707107  0.707107 )  


     PseudoPot. # 1 for C  read from file:
     ./C.BFD.upf
     MD5 check sum: 44213404c8404db625590c8337a04dfd
     Pseudo is Norm-conserving, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of 2000 points,  1 beta functions with: 
                l(1) =   0

     atomic species   valence    mass     pseudopotential
        C              4.00    12.01100     C ( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           C   tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           C   tau(   2) = (   0.3535534   0.3535534   0.3535534  )

     number of k points=     1
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

     Dense  grid:    29363 G-vectors     FFT dimensions: (  45,  45,  45)

     Estimated max dynamical RAM per process >       1.78 MB

     Estimated total dynamical RAM >      21.35 MB

     Initial potential from superposition of free atoms

     starting charge    0.63662, renormalised to    8.00000
     Starting wfcs are    8 randomized atomic wfcs

     total cpu time spent up to now is        0.1 secs

     Self-consistent Calculation

     iteration #  1     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -20.51502170 Ry
     Harris-Foulkes estimate   =     -20.80351732 Ry
     estimated scf accuracy    <       0.53086928 Ry

     iteration #  2     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  6.64E-03,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -20.54318784 Ry
     Harris-Foulkes estimate   =     -20.54962466 Ry
     estimated scf accuracy    <       0.01453145 Ry

     iteration #  3     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.82E-04,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -20.54484253 Ry
     Harris-Foulkes estimate   =     -20.54494126 Ry
     estimated scf accuracy    <       0.00041950 Ry

     iteration #  4     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  5.24E-06,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -20.54484671 Ry
     Harris-Foulkes estimate   =     -20.54486675 Ry
     estimated scf accuracy    <       0.00005990 Ry

     iteration #  5     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  7.49E-07,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485033 Ry
     Harris-Foulkes estimate   =     -20.54485046 Ry
     estimated scf accuracy    <       0.00000029 Ry

     iteration #  6     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.60E-09,  avg # of iterations =  4.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485343 Ry
     Harris-Foulkes estimate   =     -20.54485355 Ry
     estimated scf accuracy    <       0.00000170 Ry

     iteration #  7     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.60E-09,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485381 Ry
     Harris-Foulkes estimate   =     -20.54485343 Ry
     estimated scf accuracy    <       0.00000126 Ry

     iteration #  8     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.60E-09,  avg # of iterations =  3.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485329 Ry
     Harris-Foulkes estimate   =     -20.54485386 Ry
     estimated scf accuracy    <       0.00000321 Ry

     iteration #  9     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.60E-09,  avg # of iterations =  3.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485326 Ry
     Harris-Foulkes estimate   =     -20.54485339 Ry
     estimated scf accuracy    <       0.00000072 Ry

     iteration # 10     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.60E-09,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485319 Ry
     Harris-Foulkes estimate   =     -20.54485327 Ry
     estimated scf accuracy    <       0.00000022 Ry

     iteration # 11     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.80E-09,  avg # of iterations =  3.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485329 Ry
     Harris-Foulkes estimate   =     -20.54485329 Ry
     estimated scf accuracy    <       0.00000012 Ry

     iteration # 12     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-09,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -20.54485329 Ry
     Harris-Foulkes estimate   =     -20.54485329 Ry
     estimated scf accuracy    <       0.00000013 Ry

     iteration # 13     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-09,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.3 secs

     total energy              =     -20.54485329 Ry
     Harris-Foulkes estimate   =     -20.54485329 Ry
     estimated scf accuracy    <       0.00000015 Ry

     iteration # 14     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-09,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.3 secs

     total energy              =     -20.54485331 Ry
     Harris-Foulkes estimate   =     -20.54485329 Ry
     estimated scf accuracy    <       0.00000014 Ry

     iteration # 15     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-09,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.3 secs

     total energy              =     -20.54485323 Ry
     Harris-Foulkes estimate   =     -20.54485331 Ry
     estimated scf accuracy    <       0.00000025 Ry

     iteration # 16     ecut=   200.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-09,  avg # of iterations =  3.0

     total cpu time spent up to now is        0.3 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  3695 PWs)   bands (ev):

    -7.1571  15.1166  15.1166  15.1166

     highest occupied level (ev):    15.1166

!    total energy              =     -20.54485326 Ry
     Harris-Foulkes estimate   =     -20.54485327 Ry
     estimated scf accuracy    <          2.9E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =       9.51370915 Ry
     hartree contribution      =       2.88496894 Ry
     xc contribution           =      -7.39219690 Ry
     ewald contribution        =     -25.55133445 Ry

     convergence has been achieved in  16 iterations

     Writing output data file pwscf.save/

     init_run     :      0.05s CPU      0.06s WALL (       1 calls)
     electrons    :      0.15s CPU      0.18s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.02s CPU      0.02s WALL (       1 calls)
     hinit0       :      0.02s CPU      0.03s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.07s CPU      0.08s WALL (      16 calls)
     sum_band     :      0.02s CPU      0.02s WALL (      16 calls)
     v_of_rho     :      0.02s CPU      0.03s WALL (      17 calls)
     mix_rho      :      0.04s CPU      0.04s WALL (      16 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (      33 calls)
     cegterg      :      0.07s CPU      0.07s WALL (      16 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :      0.05s CPU      0.06s WALL (      47 calls)
     g_psi        :      0.00s CPU      0.00s WALL (      30 calls)
     cdiaghg      :      0.01s CPU      0.01s WALL (      46 calls)

     Called by h_psi:
     h_psi:pot    :      0.05s CPU      0.06s WALL (      47 calls)
     h_psi:calbec :      0.00s CPU      0.00s WALL (      47 calls)
     vloc_psi     :      0.05s CPU      0.06s WALL (      47 calls)
     add_vuspsi   :      0.00s CPU      0.00s WALL (      47 calls)

     General routines
     calbec       :      0.00s CPU      0.00s WALL (      47 calls)
     fft          :      0.02s CPU      0.02s WALL (      51 calls)
     ffts         :      0.00s CPU      0.01s WALL (      16 calls)
     fftw         :      0.06s CPU      0.06s WALL (     412 calls)

     Parallel routines
     fft_scatt_xy :      0.01s CPU      0.01s WALL (     479 calls)
     fft_scatt_yz :      0.03s CPU      0.03s WALL (     479 calls)

     PWSCF        :     0.24s CPU         0.29s WALL


   This run was terminated on:   9:20:29   5Nov2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
